### PR TITLE
Reviewed email IO documentation (except System Parameter descriptions).

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java [hunk 1]
```diff
@@ -52,97 +52,99 @@
...
                 @Parameter(name = "password",
-                           description = "Password of the email account.",
+                           description = "The password of the email account that is used to send emails.",
                            type = {DataType.STRING}),
                 @Parameter(name = "host",
-                           description = "Host name of the smtp server "
-                                   + "(e.g. host name for a gmail account : 'smtp.gmail.com'). The default value"
-                                   + " 'smtp.gmail.com' is only valid if email account is a gmail account.",
+                           description = "The host name of the smtp server. The email address should always be " +
+                                   "`<USERNAME>@<HOST_NAME>.com'. (e.g., If `gmail` is the host name, the " +
```
Comment (ChathurikaA) [inline component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java]: <USERNAME>@<HOST_NAME>.com is not correct.  As an example if host name for gmail account for smtp server is smtp.gmail.com

File: component/src/main/java/org/wso2/extension/siddhi/io/email/source/EmailSource.java [hunk 2]
```diff
@@ -54,116 +54,117 @@
...
                 type = {DataType.STRING },
                 optional = true,
                 defaultValue = "imap"),
         @Parameter(name = "host",
-                description = "Host name of the server "
-                        + "(e.g. host name for a gmail account with imap store : 'imap.gmail.com'). "
-                        + "The default value imap.gmail.com' is only valid if email account is a gmail account"
-                        + " with imap enable.",
+                description = "The host name of the server. The email address should always be " +
+                        "`<USERNAME>@<HOST_NAME>.com'. (e.g., If `gmail` is the host name, the " +
```
Comment (ChathurikaA) [inline component/src/main/java/org/wso2/extension/siddhi/io/email/source/EmailSource.java:85]: @<HOST_NAME>.com is not correct. As an example if host name for gmail account for imap server is imap.gmail.com

Comment (mohanvive) [reply component/src/main/java/org/wso2/extension/siddhi/io/email/source/EmailSource.java:85]: @RukshiW , can u please fix this.

Comment (RukshiW) [reply component/src/main/java/org/wso2/extension/siddhi/io/email/source/EmailSource.java:85]: Deleted my fork due to some errors. Can I do this correction via a fresh pull request?

**Comment by RukshiW** [comment]
These changes are now addressed in https://github.com/wso2-extensions/siddhi-io-email/pull/39

**Pull request closed without merging**